examples/demo_astradb.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<p>add a short description below this on what the notebook is about </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>